src/components/HtmlImport/store/htmlImport.js

```diff
@@ -1,11 +1,108 @@
-export default {
-	state: () => ({
+const getDefaultState = () => {
+	return {
+		key: 0,
 		fetchingJSON: false,
-	}),
+		settings: {
+			buttons: {
+				header: [],
+			},
+			replaceImages: {
+				displayText: "Replace image URLs",
+				value: true,
+			},
+			showReplaceSwitch: {
+				displayText: "Show a toggle button to replace images",
+				value: true,
+			},
+		},
```

Comment:
```suggestion
			replaceImages: true,
			showReplaceSwitch: true,
		},
```
Move the displayText inline